### PR TITLE
[ADMIN] Add Ideas Workshop Category admin

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: src/Entity/IdeasWorkshop/Category.php [hunk 1]
```diff
@@ -56,8 +56,13 @@ public function isEnabled(): bool
         return $this->enabled;
     }
 
-    public function setEnabled(bool $enabled): void
+    public function setEnabled(bool $enabled = false): void
```
Comment (ottaviano) [inline src/Entity/IdeasWorkshop/Category.php]: why this change?

File: src/Admin/IdeasWorkshopCategoryAdmin.php [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                'label' => 'Nom',
+                'format_title_case' => true,
+            ])
+            ->add('enabled', null, [
+                'label' => 'Visibilité',
+            ])
+        ;
+    }
+
+    /**
```
Comment (thomasbisignani) [inline src/Admin/IdeasWorkshopCategoryAdmin.php]: You can remove this PHPDoc

File: src/Admin/IdeasWorkshopCategoryAdmin.php [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    {
+        $datagridMapper
+            ->add('name', null, [
+                'label' => 'Nom',
+                'show_filter' => true,
+            ])
+        ;
+    }
+
+    /**
```
Comment (thomasbisignani) [inline src/Admin/IdeasWorkshopCategoryAdmin.php]: Same here

File: src/Entity/IdeasWorkshop/Category.php (hunk 1 above)
Comment (illusionOfParadise) [reply src/Entity/IdeasWorkshop/Category.php]: No reason. Removed.

File: src/Admin/IdeasWorkshopCategoryAdmin.php (hunk 3 above)
Comment (illusionOfParadise) [reply src/Admin/IdeasWorkshopCategoryAdmin.php]: Done.

File: src/Admin/IdeasWorkshopCategoryAdmin.php (hunk 2 above)
Comment (illusionOfParadise) [reply src/Admin/IdeasWorkshopCategoryAdmin.php]: Done.

File: src/Entity/IdeasWorkshop/Category.php [hunk 4]
```diff
@@ -60,4 +64,19 @@ public function setEnabled(bool $enabled): void
     {
         $this->enabled = $enabled;
     }
+
+    public function getName(): ?string
+    {
+        return $this->name;
+    }
+
+    public function setName(string $name)
```
Comment (ottaviano) [inline src/Entity/IdeasWorkshop/Category.php]: `: void`
